extensions/compose/src/extension.ts

```diff
@@ -31,6 +31,11 @@ import { ComposeInstallation } from './installation';
 let composeExtension: ComposeExtension | undefined;
 let composeVersionMetadata: ComposeGithubReleaseArtifactMetadata | undefined;
 
+const composeCliName = 'docker-compose';
+const composeDisplayName = 'Compose';
+const composeDescription = `Compose is a specification for defining and running multi-container applications. We support both [podman compose](https://docs.podman.io/en/latest/markdown/podman-compose.1.html) and [docker compose](https://github.com/docker/compose) commands.\n\nMore information: [compose-spec.io](https://compose-spec.io/)`;
```

Comment:
hello, I'm not sure about the part around `docker compose`

For sure you can run `docker-compose` CLI but `docker compose` if you don't have `docker` CLI or compose being at a right place it'll not work

Reply:
here I think you should use @mairin text from https://github.com/containers/podman-desktop/pull/4534 ?